packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -1,18 +1,284 @@
 <template>
 	<tera-drilldown :title="node.displayName" @on-close-clicked="emit('close')">
-		<section>TODO</section>
+		<section :tabName="ConfigTabs.Wizard">
+			<tera-drilldown-section>
+				<div>
+					<Steps :model="formSteps" :readonly="false" @update:active-step="activeIndex = $event" />
+				</div>
+				<div v-if="activeIndex === 0" class="form-section">
+					<h3>Name</h3>
+					<InputText
+						placeholder="Enter a name for this configuration"
+						v-model="configName"
+						@update:model-value="() => debouncedUpdateState({ configName })"
+					/>
+					<h3>Description</h3>
+					<Textarea
+						placeholder="Enter a description"
+						v-model="configDescription"
+						@update:model-value="() => debouncedUpdateState({ configDescription })"
+					/>
+				</div>
+				<div v-else-if="activeIndex === 1">
+					<tera-model-config-editor
+						v-if="model"
+						:model="model"
+						:initials="configInitials"
+						:parameters="configParams"
+						@update-param="updateConfigParam"
+						@update-initial="updateConfigInitial"
+					/>
+				</div>
+			</tera-drilldown-section>
+		</section>
+		<section :tabName="ConfigTabs.Notebook">
+			<h4>TODO</h4>
+		</section>
+		<template #preview>
+			<tera-drilldown-preview
+				title="Output"
+				:options="outputs"
+				v-model:output="selectedOutputId"
+				@update:output="onUpdateOutput"
+				@update:selection="onUpdateSelection"
+				is-selectable
+			>
+				<div v-if="configCache[selectedConfigId]">
+					<TeraModelSemanticTables
+						:model="configCache[selectedConfigId].configuration"
+						:is-editable="false"
+					/>
+				</div>
```

Comment:
No need for div also component should be kebab case.

Reply:
done